packages/reports/addon/components/filter-values/dimension-date-range.js

```diff
@@ -1,67 +1,65 @@
 /**
- * Copyright 2019, Yahoo Holdings Inc.
+ * Copyright 2020, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  *
  * Usage:
- *   {{filter-values/dimension-date-range
- *       filter=filter
- *       onUpdateFilter=(action 'update')
- *   }}
+ *   <Filter::Values::DimensionDateRange
+ *     @filter={{this.filter}}
+ *     @onUpdateFilter={{this.update}}
+ *   />
  */
 
-import { oneWay } from '@ember/object/computed';
 import Component from '@ember/component';
-import { get } from '@ember/object';
-import layout from '../../templates/components/filter-values/dimension-date-range';
+import { get, action } from '@ember/object';
+import { oneWay } from '@ember/object/computed';
 import Moment from 'moment';
+import layout from '../../templates/components/filter-values/dimension-date-range';
+import { layout as templateLayout, tagName } from '@ember-decorators/component';
 
-export default Component.extend({
-  layout,
-
-  /**
-   * @property {Array} classNames
-   */
-  classNames: ['filter-values--dimension-date-range-input'],
-
+@templateLayout(layout)
+@tagName('')
+class DimensionDateRange extends Component {
```

Comment:
I think there might have been an update to ember-decorators so we can `export default` at the top again